database/migrations/2014_10_12_000000_create_users_table.php

```diff
@@ -14,10 +14,8 @@ public function up()
     {
         Schema::create('users', function (Blueprint $table) {
             $table->increments('id');
```

Comment:
@angaither Ok, so seems like we need the app id that's an integer for the pivot table stuff that relies on ids to do the associations, otherwise the migrations and artisan yell at me a lot...